### PR TITLE
ncnews-3, GET topics endpoint complete with passing test

### PR DESCRIPTION
**Pull request by philjacks**
- endpoint for getting topics added
- used TDD during development
- controllers & models folders added
- express & supertest used

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+const express = require("express");
+const { getTopics } = require("./controllers/getControllers");
+
+const app = express();
+
+app.use(express.json());
```
Comment (Augs0) [inline app.js:6]: Do we need this yet? :p

File: listen.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+const app = require("./app");
+const PORT = 9090;
+
+app.listen(PORT, () => {
```
Comment (Augs0) [inline listen.js:4]: We don't need an app,listen yet

File: controllers/getControllers.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+const { fetchTopicsFromDb } = require("../models/getModels");
+
+exports.getTopics = (req, res) => {
+  fetchTopicsFromDb().then((topics) => {
+    res.status(200).send({ topics });
+  });
```
Comment (Augs0) [inline controllers/getControllers.js:6]: You will probably want some kind of catch block here to achieve error handling

**Review by Augs0 — CHANGES_REQUESTED** [review]
Just a few points to consider before moving on. I would also look at the names of your controllers and models. It might make a bit more sense organisationally to go by the parts of your app e.g. topics rather than the http request e.g. GET

File: app.js (hunk 1 above)
Comment (philjacks) [reply app.js]: Removed :)

File: listen.js (hunk 2 above)
Comment (philjacks) [reply listen.js]: Removed :)

File: controllers/getControllers.js (hunk 3 above)
Comment (philjacks) [reply controllers/getControllers.js]: Done :)

**Pull request merged**